engine/src/main/resources/assets/i18n/menu_id.lang

```diff
@@ -0,0 +1,255 @@
+    "nui-editor-save": "Simpan sebagai...",
+    "nui-editor-settings-title": "Pengaturan Editor",
+    "nui-editor-undo": "Undo",
+    "oculus-rift-support": "Dukungan Oculus Rift",
+    "online-players": "Pemain Online",
+    "open-inventory": "Buka inventaris";
+    "outline": "Garis besar",
+    "particle-effect-limit": "Batasi limit partikel",
+    "parallax-mapping": "Pemetaan Parallax",
+    "parsing-content": "Mengurai content ..",
+    "player-color": "Warna Pemain",
+    "player-name": "Nama Pemain",
+    "player-height": "Tinggi Pemain",
+    "player-eye-height": "Tinggi Mata Pemain",
+    "player-settings": "Pemain",
+    "player-settings-title": "Pengaturan Pemain",
+    "please-wait": Mohon menunggu..",
+    "preview-world-title": "Pratinjau Dunia...",
+    "preview-zoom-factor": "Zoom",
+    "previous-toolbar-item": "Barang Toolbar sebelumnya",
+    "remove-binding": "Hilangkan",
+    "remove-server": "Hilangkan",
+    "reset-default": "Atur ulang",
+    "reset-fov-amount": "Atur ulang banyak FoV",
+    "resolution-scale": "Skala Resolusi",
+    "respawn": "Respawn",
+    "restore-defaults": "Kembalikan seperti semula",
+    "return-main-menu": "Keluar ke Menu Utama",
+    "rotation-dead-zone": "Putaran Sumbu Mati",
+    "save-game-path": "Simpan path permainan:",
+    "screenshot-format": "Format screenshot",
+    "screenshot-size": "Ukuran screenshot",
+    "screenshot-size-double": "Ukuran dua kali lipat",
+    "screenshot-size-half": "Ukuran setengah",
+    "screenshot-size-normal": "Ukuran normal",
+    "screenshot-size-quarter": "Ukuran seperempat",
+    "select-game-title": "Pilih Permainan",
+    "select-modules": "Module...",
+    "select-modules-title": "Pilih Modul...",
+    "select-multiplayer-game-sub-title": "Multiplayer",
+    "select-singleplayer-game-sub-title": "Singleplayer",
+    "select-world-gen": "Pilih Generator Dunia",
+    "select-world-gen-info": "mengaktifkan modul mungkin akan menambahkan",
+    "server-address": "Alamat Server",
+    "server-list-complete": "Mendaftarkan server selesai.",
+    "server-name": "Nama",
+    "server-owner": "Pemilik",
+    "server-port": "Port",
+    "settings-language": "Bahasa",
+    "settings-title": "Pengaturan",
+    "shadows": "Bayangan",
+    "shadows-off": "Mati",
+    "shadows-on": "Nyala",
+    "shadows-pcr": "Nyala (PCR)",
+    "sound-volume": "Volume Suara",
+    "ssao": "SSAO",
+    "start-game": "Play",
+    "start-singleplayer-game": "Singleplayer",
+    "this-language-English": "Inggris",
+    "this-language-native": "Inggris",
+    "update-module": "Memperbarui",
+    "video-display-mode": "Mode Display",
+    "video-fullscreen": "Layar Penuh",
+    "video-windowed-fullscreen": "Layar Penuh Windowed",
+    "video-windowed": "Windowed",
+    "video-graphic-settings": "Window / Pengaturan Grafik",
+    "video-high-end-settings": "High-End / Pengaturan Eksperimental",
+    "video-preset": "Preset",
+    "video-preset-custom": "Custom",
```

Comment:
Medium and Minimal is a commonly used word in Indonesia (Only different pronounciation) Although there's an alternative translation, it's rarely used. Custom will be replaced to Kustom.